src/app/tests/suites/certification/ci-pics-values

```diff
@@ -1 +1,82 @@
+BLE=1
```

Comment:
Can we have some check in CI that fails if there is a PICS value in PICS.yaml that is not listed here?  Followup is OK for this.

Reply:
Created an issue: https://github.com/project-chip/connectedhomeip/issues/16735